bindings/python/examples/simple_transaction/request_funds.py

```diff
@@ -20,8 +27,10 @@
 print(
     f'balance before faucet request: { balance[ "baseCoin" ][ "available" ] }')
 
+addresses = account.generate_addresses(1)
+
 FAUCET_URL = os.environ.get('FAUCET_URL', 'https://faucet.testnet.shimmer.network/api/enqueue')
-response = Client().request_funds_from_faucet(FAUCET_URL, "rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu")
+response = Client().request_funds_from_faucet(FAUCET_URL, address=addresses[0]['address'])
```

Comment:
If you rebase you can now use `wallet.client()` and don't need the Client import anymore

Reply:
Done